tests/testapp/migrations/0003_alter_customdocument_options_alter_video_options.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 5.0.4 on 2024-04-05 16:08
```

Comment:
nitpick (or personal preference) - renegerate the tests in `0001_initial`. While minor, running tests will be a tad speedier

Reply:
No problem. Updated in [2ded683](https://github.com/torchbox/wagtail-bynder/pull/19/commits/2ded683fd9a029de7a9eb0f573dfa3595b493c1a)